test/e2e/component_test.go

```diff
@@ -18,13 +17,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
-	retryInterval        = time.Second * 5
```

Comment:
I don't see these constants defined anywhere else. And it is used in `TestComponent`.

Reply:
https://github.com/redhat-developer/devconsole-operator/blob/master/test/e2e/custom_resource_operator_test.go#L25

Reply:
That file was removed in Avni's PR #33 But looks like it got introduced again from PR #55

Reply:
We should remove that file, and just keep `component_test.go`. It is the same test.

Reply:
Fixed

Reply:
AH yes!